classes/classes/Player.as

```diff
@@ -2575,10 +2567,10 @@ use namespace kGAMECLASS;
 			if (isNaga()) maxSpe += 10;
 			if (isTaur() || isDrider()) maxSpe += 20;
 			//Apply New Game+
-			maxStr += 25 * newGamePlusMod();
-			maxTou += 25 * newGamePlusMod();
-			maxSpe += 25 * newGamePlusMod();
-			maxInt += 25 * newGamePlusMod();
+			maxStr += ascensionFactor();
+			maxTou += ascensionFactor();
+			maxSpe += ascensionFactor();
+			maxInt += ascensionFactor();
```

Comment:
The change here is mostly to give you an example, how `public function ascensionFactor(multiplier:Number = 25):Number` works.